db/migrate/20230923135510_create_tasks.rb

```diff
@@ -0,0 +1,13 @@
+class CreateTasks < ActiveRecord::Migration[6.1]
+  def change
+    create_table :tasks do |t|
+      t.string :name, null: false
+      t.text :description
+      t.string :status, null: false
+      t.datetime :deadline
```

Comment:
Usually the date and time fields should be named with the _at postfix

```suggestion
      t.datetime :deadline_at
```